Updated Lob API

shrav opened the pull request:
Updated Lob description, links, and then!

shrav committed:
Update content.json

JonHMChan commented:
I'm going to have to reject this PR until there are some edits:

**Rule 1**: "Links should...take the user immediately to the relevant information (i.e. for html go to the html exercise in codecademy, rather than codeacademy's home page - the user should not have to do any searching to start learning)" - the link provided is neither a tutorial nor does it take them immediately relevant learning information. Feel free to add in a new link if it is a walkthrough or learning resource only.
**Rule 4**: "All names must be lowercase (even in the $then fields and descriptions)"

Not yet a rule, but the description should also make explicit _what_ the technology is (we cannot assume the user knows). Say that it is "An API that lets you print and mail documents, postcards, photos, and more" or along those lines.

I'll wait to merge pending those considerations.

shrav committed:
Added changes requested

shrav commented:
@JonHMChan I updated the lower case names and changes the description. Also I removed registration, but I thought it was something that was okay to add, since right above it there is a link for another API as so:

```
  {
    "name": "Get a free API key",
    "level": 1,
    "url": "http://www.alchemyapi.com/api/register.html"
  },
```

JonHMChan commented:
Thanks for pointing it out on the other - I'll remove. Merged.
